esp-hal/src/uart.rs

```diff
@@ -507,6 +520,22 @@ where
         self.tx.write_bytes(data)
     }
 
+    /// Change interpretation of line polarity for inputs/outputs
+    pub fn set_line_invert(&mut self, line: UartLine, invert: bool) {
+        T::register_block().conf0().write(|w| match line {
+            UartLine::Rxd => w.rxd_inv().bit(invert),
+            UartLine::Txd => w.txd_inv().bit(invert),
+            #[cfg(not(any(esp32c6, esp32h2)))]
```

Comment:
inverting these pins should be available via conf1 for C6/H2

Reply:
You're right, fixed in latest